packages/amplication-client/src/Workspaces/WorkspaceFooter.tsx

```diff
@@ -86,13 +102,23 @@ const WorkspaceFooter: React.FC<{}> = () => {
               className={`${CLASS_NAME}__github-icon`}
             />
             <GitRepoDetails />
-            <a
-              className={`${CLASS_NAME}__gh-link`}
-              href={gitRepositoryUrl}
-              target="github"
-            >
-              Open With GitHub
-            </a>
+            {lastResourceBuild && githubUrl ? (
+              <a
+                className={`${CLASS_NAME}__gh-link`}
+                href={githubUrl}
+                target="github"
+              >
+                Open With GitHub
+              </a>
+            ) : (
+              <a
+                className={`${CLASS_NAME}__gh-link`}
+                href={gitRepositoryUrl}
+                target="github"
+              >
+                Open With GitHub
+              </a>
+            )}
```

Comment:
use only one <a> and make sure that value returned from githubUrl is equal to githubUrl when needed